crates/shared/src/arguments.rs

```diff
@@ -184,6 +184,10 @@ pub struct Arguments {
     #[clap(long, env)]
     pub blocknative_api_key: Option<String>,
 
+    /// Driver URL for gas price estimation when using the Driver estimator.
+    #[clap(long, env)]
+    pub gas_estimation_driver_url: Option<String>,
```

Comment:
This should probably be an Url, not String.
```suggestion
    pub gas_estimation_driver_url: Option<Url>,
```